index.js

```diff
@@ -68,7 +71,34 @@ function toggleColor(element) {
   }
 }
 
+element.addEventListener('click', function(){
+  toggleColor(element)
+})
 /***** Deliverable 2 *****/
 
+const like = document.querySelector('.like-button')
+
+like.addEventListener('click',function(){
+  const likesCount = document.querySelector('.likes')
+  const likesValue = parseInt(likesCount.textContent.split(' ')[0]) + 1
+  likesCount.textContent = `${likesValue} ${likesCount.textContent.split(' ')[1]}`
```

Comment:
Your solution looks like it'll work! Another way do solve this would be grabbing the integer from the likesCount and incrementing it just using `parseInt(likesCount) + 1`. That will grab the number from the string and turn it into an integer. Then for updating `likesCount.textContent`, you could write: 

```javascript
likesCount.textContent = `${likesValue} likes`
```